auth_totp/README.rst

```diff
@@ -0,0 +1,103 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Let's do LGPL-3

Future reference, if we aren't inheriting anything and we are designing it inside of Odoo - I want LGPL always. If we are inheriting, we need to use AGPL if there is an AGPL in the inheritance chain.

Outside of Odoo, MIT if possible. Licensing sucks dude, (dealwithit)